hpack/src/main/java/com/twitter/hpack/Decoder.java

```diff
@@ -271,6 +287,7 @@ public void decode(InputStream in, HeaderListener headerListener) throws IOExcep
         }
 
         name = readStringLiteral(in, nameLength);
+        nameString = new String(name, HpackUtil.ISO_8859_1);
```

Comment:
would prefer not to hardcode this here and instead allow caller to pass a cached representation from addHeader()